cliqz_env.sh

```diff
@@ -55,34 +55,43 @@ if [ "$CQZ_RELEASE_CHANNEL" = "release" ]; then
   export MOZ_UPDATE_CHANNEL=release
   # turn on PGO only for Release Windows build
   if [ $IS_WIN ]; then
-    if [ $CQZ_BUILD_ID ]; then
-      export MOZ_PGO=1 # release build optimization flag
-    fi
+    export MOZ_PGO=1 # release build optimization flag
   fi
 fi
 
 export MOZ_OBJDIR=../obj
 
-export MOZCONFIG=browser/config/cliqz-release.mozconfig
+if [ "$CQZ_BUILD_64BIT_WINDOWS" = "1" ]; then
```

Comment:
shouldn't this be `==` ?

Reply:
this should be the same in this case: 

[ STRING1 == STRING2 ] | True if the strings are equal.  "=" may be used instead of "==" for strict POSIX compliance.

http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html

Reply:
I did this in the same way, as I did before with checking for "release" channel (in same script). Can change everywhere to ==, let me check more carefully